src/test/scala/GuessWhoGameSpec/BoardSpec.scala

```diff
@@ -0,0 +1,149 @@
+package GuessWhoGameSpec
+
+import GuessWhoGame._
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers._
+
+class BoardSpec extends AnyFlatSpec {
+
+  val characters: List[Character] = List(
+    Character("joe", "male", "brown", "blue", wearsGlasses = false, facialHair = false),
+    Character("muhammad", "male", "black", "brown", wearsGlasses = true, facialHair = true),
+    Character("april", "female", "blonde", "blue", wearsGlasses = true, facialHair = false),
+    Character("sally", "female", "blue", "green", wearsGlasses = true, facialHair = true),
+    Character("spencer", "male", "brown", "brown", wearsGlasses = false, facialHair = true),
+    Character("gemma", "female", "ginger", "blue", wearsGlasses = true, facialHair = false),
+    Character("jamie", "male", "none", "green", wearsGlasses = true, facialHair = true),
+    Character("jessica", "female", "green", "green", wearsGlasses = false, facialHair = false),
+    Character("bilal", "male", "brown", "green", wearsGlasses = false, facialHair = true),
+    Character("lisa", "female", "black", "brown", wearsGlasses = true, facialHair = false),
+    Character("tom", "male", "brown", "blue", wearsGlasses = true, facialHair = false),
+    Character("cheryl", "female", "blonde", "green", wearsGlasses = false, facialHair = false),
+    Character("arei", "male", "brown", "brown", wearsGlasses = false, facialHair = true),
+    Character("kelly", "female", "none", "green", wearsGlasses = false, facialHair = false),
+    Character("tayamul", "male", "brown", "blue", wearsGlasses = false, facialHair = false),
+    Character("elaine", "female", "ginger", "blue", wearsGlasses = false, facialHair = true),
+    Character("roshan", "male", "brown", "blue", wearsGlasses = true, facialHair = false),
+    Character("patricia", "female", "purple", "yellow", wearsGlasses = false, facialHair = false),
+    Character("dave", "male", "none", "yellow", wearsGlasses = false, facialHair = false),
+    Character("bobbiana", "female", "brown", "blue", wearsGlasses = true, facialHair = false)
+  )
+
+  //prints out characters onto the board
+  "gameBoard" should "initialise with the provided characters" in {
+    val gameBoard = new Board(characters)
+    gameBoard.getRemainingCharacters shouldEqual characters
+  }
+
+  // Selects a character
+  it should "select a random character at the start of the game and keeps this character in the list" in {
```

Comment:
What is "it"?
Remember, make your code readable. Think about nesting the tests if you are writing multiple about the game board?